client/src/components/Filters/FilterSalary.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Slider from "@mui/material/Slider";
+import Box from "@mui/material/Box";
+
+export default function SalarySliderSteps({ handleSalary }) {
+  const [sliderValue, setSliderValue] = useState(0);
+
+  return (
+    <Box sx={{ width: 200, color: "green" }}>
+      <Slider
```

Comment:
It might be worth either increasing this to go from 10 to 250000, or have a description that states that the min, max, and step values are multiplied by 1,000